File: 240901/원소 10개의 합/sum-of-10-elements.java
import java.util.Scanner;
public class Main {
    public static void main(String[] args) {
        Scanner sc=new Scanner(System.in);
        int []arr=new int[10];
        int sum_val=0;
        for(int i=0; i<10; i++){
            arr[i]=sc.nextInt();
            sum_val+=arr[i];
        }

        System.out.printf("%d",sum_val);
        // 여기에 코드를 작성해주세요.
    }
}

File: 240901/원소 10개의 합/sum-of-10-elements.c
#include <stdio.h>

int main() {
    int arr[10];
    int sum_val=0;

    for(int i=0; i<10; i++){
        scanf("%d",&arr[i]);
        sum_val+=arr[i];
    }
    printf("%d",sum_val);
    // 여기에 코드를 작성해주세요.
    return 0;
}